tests/integration/telemetry/tracing/tracing.go

```diff
@@ -18,28 +18,41 @@ package tracing
 import (
 	"fmt"
 	"testing"
+	"time"
 
-	"istio.io/istio/pkg/test/framework/components/bookinfo"
+	"istio.io/istio/pkg/config/protocol"
+	"istio.io/istio/pkg/test/framework/components/echo"
+	"istio.io/istio/pkg/test/framework/components/echo/echoboot"
 	"istio.io/istio/pkg/test/framework/components/istio"
 	"istio.io/istio/pkg/test/framework/components/istio/ingress"
 	"istio.io/istio/pkg/test/framework/components/namespace"
 	"istio.io/istio/pkg/test/framework/components/zipkin"
 	"istio.io/istio/pkg/test/framework/resource"
+	"istio.io/istio/pkg/test/util/retry"
 )
 
 var (
+	client, server echo.Instances
 	ist            istio.Instance
-	bookinfoNsInst namespace.Instance
 	ingInst        ingress.Instance
 	zipkinInst     zipkin.Instance
+	appNsInst      namespace.Instance
+)
+
+const (
+	// For multicluster tests we multiply the number of requests with a
+	// constant multiplier to make sure we have cross cluster traffic.
+	requestCountMultipler = 3
```

Comment:
can we share this in a common place (i think i recall this being used for similar purpose elsewhere) ?

Reply:
Agreed.. Done